### PR TITLE
Added feedback for toggling FIPM, updated ReadMe

### LINKED ISSUE #519: Reporting the state of free item positioning mode
Free item positioning mode drastically changes how Reaper behaves. Ergo, it would be beneficial to have this reported in two ways:

1. Feedback for the following toggle:
Action: Track properties: Toggle free item positioning
ID: 40641

2. Can we be notified if FIPM is on when moving to a track?

As ever, thanks in advance.

### PR DESCRIPTION
**Pull request by ScottChesworth**
Also fixed a typo in the first paragraph of the ReadMe while I was there.
Fixes #519.

### CODE REVIEW

**Comment by jcsteh** [comment]
Is this ready for review or are you planning to add the track nav reporting here too? Either way, let me know when you want me to review it.

**Comment by ScottChesworth** [comment]
Hold fire on this one, I'm gonna take a shot at track nav reporting. The other PR I just sent that fixes inaccurate action feedback when adjusting velocity in the piano roll is ready to go though.

**Comment by ScottChesworth** [comment]
Think this is ready for review now @jcsteh.
Pushing the track nav report got a bit hairy, so apologies if I've made a mess.

File: src/reaper_osara.cpp [hunk 1]
```diff
@@ -3,6 +3,7 @@
  * Main plug-in code
  * Author: James Teh <jamie@jantrid.net>
  * Copyright 2014-2019 NV Access Limited, James Teh
+ * Copyright 2019-2021 James Teh
```
Comment (jcsteh) [inline src/reaper_osara.cpp:6]: Strictly speaking, having multiple copyright entries is probably more accurate, but I think that's going to get unmanageable with time, so I'm condensing.

```suggestion
 * Copyright 2014-2021 NV Access Limited, James Teh
```

**Review by jcsteh — APPROVED** [review]
Thanks! Great work!

The commits got a bit messy I think because of the other branch. All fixed.

File: src/reaper_osara.cpp [hunk 2]
```diff
@@ -441,6 +441,10 @@ bool isItemSelected(MediaItem* item) {
 	return *(bool*)GetSetMediaItemInfo(item, "B_UISEL", NULL);
 }
 
+bool isFreeItemPositioningEnabled(MediaTrack* track) {
+	return *(int*)GetSetMediaTrackInfo(track, "B_FREEMODE", NULL);
```
Comment (jcsteh) [inline src/reaper_osara.cpp:445]: Ah crap, I missed this. That should be `bool*`, not `int*`, but it doesn't really matter. I should go to bed apparently.

**Pull request merged**